CoreFoundation/Base.subproj/CFAvailability.h

```diff
@@ -135,13 +135,13 @@
 #define __CF_ENUM_FIXED_IS_AVAILABLE (__cplusplus && __cplusplus >= 201103L && (__has_extension(cxx_strong_enums) || __has_feature(objc_fixed_enum))) || (!__cplusplus && (__has_feature(objc_fixed_enum) || __has_extension(cxx_fixed_enum)))
 
 #if __CF_ENUM_FIXED_IS_AVAILABLE
-#define __CF_NAMED_ENUM(_type, _name)     int __CF_ENUM_ ## _name; enum __CF_ENUM_ATTRIBUTES _name : _type; typedef enum _name _name; enum _name : _type
+#define __CF_NAMED_ENUM(_type, _name)     enum __CF_ENUM_ATTRIBUTES _name : _type _name; enum _name : _type
 #define __CF_ANON_ENUM(_type)             enum __CF_ENUM_ATTRIBUTES : _type
-#define CF_CLOSED_ENUM(_type, _name)      int __CF_ENUM_ ## _name; enum __CF_CLOSED_ENUM_ATTRIBUTES _name : _type; typedef enum _name _name; enum _name : _type
+#define CF_CLOSED_ENUM(_type, _name)      enum __CF_CLOSED_ENUM_ATTRIBUTES _name : _type _name; enum _name : _type
 #if (__cplusplus)
 #define CF_OPTIONS(_type, _name) _type _name; enum __CF_OPTIONS_ATTRIBUTES : _type
 #else
-#define CF_OPTIONS(_type, _name) int __CF_OPTIONS_ ## _name; enum __CF_OPTIONS_ATTRIBUTES _name : _type; typedef enum _name _name; enum _name : _type
+#define CF_OPTIONS(_type, _name) enum __CF_OPTIONS_ATTRIBUTES _name : _type _name; enum _name : _type
```

Comment:
I had to revert these macro patches when building for Android, buttaface/swift-android-sdk@dba89d2, or [building CoreFoundation fails with this error](https://github.com/buttaface/swift-android-sdk/runs/3608214051?check_suite_focus=true#step:7:2268):
```
error: non-defining declaration of enumeration with a fixed underlying type is only permitted as a standalone declaration; missing list of enumerators? [-Welaborated-enum-base]
```
I checked the preprocessed source produced with and without these macro changes and got the following diff:
```
--- a/cf.i	2021-09-16 07:54:13.941791291 +0000
+++ b/cf.i	2021-09-16 08:08:32.447061538 +0000
@@ -199,7 +199,7 @@
 typedef __attribute__((objc_bridge(id))) CFTypeRef CFPropertyListRef;
 
 
-typedef int __CF_ENUM_CFComparisonResult; enum __attribute__((enum_extensibility(open))) CFComparisonResult : CFIndex; typedef enum CFComparisonResult CFComparisonResult; enum CFComparisonResult : CFIndex {
+typedef enum __attribute__((enum_extensibility(open))) CFComparisonResult : CFIndex CFComparisonResult; enum CFComparisonResult : CFIndex {
     kCFCompareLessThan = -1L,
     kCFCompareEqualTo = 0,
     kCFCompareGreaterThan = 1
```
I don't know what to make of that C gibberish so if anybody else knows how to fix it, I'd appreciate it.